internal/command/postgres/attachments.go

```diff
@@ -0,0 +1,102 @@
+package postgres
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/superfly/flyctl/client"
+	"github.com/superfly/flyctl/internal/app"
+	"github.com/superfly/flyctl/internal/command"
+	"github.com/superfly/flyctl/internal/config"
+	"github.com/superfly/flyctl/internal/flag"
+	"github.com/superfly/flyctl/internal/render"
+	"github.com/superfly/flyctl/iostreams"
+)
+
+func newAttachments() *cobra.Command {
+	const (
+		short = "manage database attachments to apps"
+		long  = short + "\n"
+		usage = "attachments [command]"
+	)
+
+	cmd := command.New(usage, short, long, nil)
+
+	cmd.AddCommand(
+		newAttachmentsList(),
+		newAttach(),
```

Comment:
Do you think we should rename the attach/detach sub commands with this? 

```
fly pg attachments attach` doesn't feel quite right.

Reply:
@davissp14 I think people are too used to it and it makes sense. I'd rather find a new name for the new attachments command.